src/test/java/io/fabric8/elasticsearch/plugin/OpenshiftRequestContextFactoryTest.java

```diff
@@ -103,9 +103,14 @@ private void assertKibanaIndexIs(String mode) {
         switch (mode) {
         case "shared_ops":
             assertEquals("Exp. Kibana index mode to to be '.kibana'",
-                    ConfigurationSettings.DEFAULT_USER_PROFILE_PREFIX,
+                    ConfigurationSettings.DEFAULT_USER_PROFILE_PREFIX + "_ops",
```

Comment:
This change will have ramifications introduced by earlier versions of the plugin for existing clusters where '.kibana' is for opts.  It will mean on upgrade their visualizations, searches, and dashboards would be lost to them.

Reply:
you are right so maybe it is a better way to let the index for ops to ".kibana" and add a new index ".kibana_user" or something like that, do you have a proposal?